tests/ProjectStorageBackendTest.php

```diff
@@ -90,6 +92,65 @@ public function testProjectStorageAssignBackend(string $backendName): void
         $this->client->purgeDeletedProject($project['id']);
     }
 
+    public function testProjectStorageAssignBackendFailedWithNonNumericBackendId(): void
+    {
+        $name = 'My org';
+        $organization = $this->client->createOrganization($this->testMaintainerId, [
+            'name' => $name,
+        ]);
+
+        $project = $this->client->createProject($organization['id'], [
+            'name' => 'My test',
+            'dataRetentionTimeInDays' => 1,
+        ]);
+
+        $client = new GuzzleClient([
+            'base_uri' => getenv('KBC_MANAGE_API_URL'),
+        ]);
+
+        $requestOptions = [
+            'headers' => [
+                'X-KBC-ManageApiToken' => getenv('KBC_MANAGE_API_TOKEN'),
+                'Accept-Encoding' => 'gzip',
+                'Content-Type' => 'application/json',
+                'User-Agent' => 'Keboola Manage API PHP Client',
+            ],
+        ];
+
+        try {
+            $requestOptions['body'] = '{"storageBackendId": "non-numeric"}';
+            $client->post('/manage/projects/' . $project['id'] . '/storage-backend', $requestOptions);
+            $this->fail('Should fail with 400');
+        } catch (GuzzleException $e) {
+            $this->assertEquals(400, $e->getCode());
+            $this->assertStringContainsString('storageBackendId must be an integer.', $e->getMessage());
+        }
+
+        $this->client->removeProjectStorageBackend($project['id'], $project['backends']['snowflake']['id']);
```

Comment:
Toto ma este dve nevyhody, keby sme sa rozhodli z nejakeho dovodu ze nam budu bezat manage testy nad inym backendom tak to musime upravit. Alebo teoreticky ak na testingu nema ten maintainer default backend tak to bude null a spadne to.

Co si pre ten test zalozi vlastneho mainstainera? Je to len metadatova operacia:
-  zalozis noveho maintainera a nenastavis mu ziadny default backend
- potom nemusis mazat projectbackend, ale len ho priradis.

pred kazdy testom sa automaciky premazavaju maintainery ak to neieje ten tvoj hlavny https://github.com/keboola/kbc-manage-api-php-client/blob/c5decb33af65a73aec461b2bd61360dab69b37a5/tests/ClientTestCase.php#L194-L207 takze nemusis robit ziadny cleaup.

Reply:
good!

Reply:
done